assets/sass/admin.scss

```diff
@@ -139,3 +139,4 @@
 @import "components/settings/googlesitekit-settings-module-warning";
 @import "components/settings/googlesitekit-settings-module-header";
 @import "components/settings/googlesitekit-settings-notice";
+@import "components/wp-dashboard/googlesitekit-noscript";
```

Comment:
`.no-js` appears in the body so it would target the admin bar too.

Reply:
> `.no-js` appears in the body so it would target the admin bar too.

In wp-admin, yes but not on the front end when the user is logged in which is why this is needed in the adminbar stylesheet because Site Kit's `admin.css` isn't loaded there.